commit-message.md

```diff
@@ -0,0 +1,36 @@
+# Guidelines to create great commit messages
+
+
+We use [GitCop](https://gitcop.com) to check that commit messages are
+properly written. The rules are the following:
+
+* The first line of a commit message, called the subject line should
+  not be more than 80 characters long.
+
+* The commit message should end with the following trailers:
+
+  ```
+  Licence: MIT
+  Signed-off-by: User Name <email@address>
+  ```
+
+  where "User Name" is the author's real name and email@address one of
```

Comment:
On Sun, Jun 07, 2015 at 01:30:40PM -0700, Juan Batiz-Benet wrote:
> Don't think we should go the "required signed pgp commit" route just
> yet…

Right, I was just suggesting that as an option for people who were
wondering “how do I convince a court that I am the pseudonymous author
of this commit”.